text/char-rnn/char-rnn.jl

```diff
@@ -1,59 +1,80 @@
+# # Character-level Recurrent Neural Network
+#- Train model on Shakespeare's works
+#- Have model write like Shakespeare at the end
+
+# # 1. Import Dependencies
+
 using Flux
 using Flux: onehot, chunk, batchseq, throttle, crossentropy
 using StatsBase: wsample
 using Base.Iterators: partition
 
+# # 2. Data Download & Pre-processing
+# - Source of data: Shakespeare text from https://cs.stanford.edu/people/karpathy/char-rnn/
+# - Generate character tokens
+# - Partition in batches for input
```

Comment:
Probably good to talk about how these batches are divided. The comments in the code blocks would do well as Markdown cells that detail the why and how in a couple lines

Reply:
Sure! I'll add these details in a few days' time if you don't mind, it's actually testing/examination period for my school!